### PR TITLE
Support node v15

### LINKED ISSUE #16: Cannot install console-testing-library@0.5.0 on node ^15
```
error console-testing-library@0.5.0: The engine "node" is incompatible with this module. Expected version "^10 || ^12 || ^13 || ^14". Got "15.2.1"
```

It seems that `console-testing-library@0.4.0` is fine

### PR DESCRIPTION
**Pull request by csi-lk**
As discussed in #16 tiny change to fix node v15 support

Resolves #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.5.0",
   "description": "Testing console the right way",
   "engines": {
-    "node": "^10 || ^12 || ^13 || ^14"
+    "node": "^10 || ^12 || ^13 || ^14 || ^15"
```
Comment (kevin940726) [inline package.json:6]: Thanks for the PR! I'm thinking we should probably loosen the guards. Maybe just:

```suggestion
    "node": "^10 || >=12"
```

**Review by kevin940726 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (csi-lk) [reply package.json]: Yeah good idea, pushed an update with your suggestion 👍

**Review by kevin940726 — APPROVED** [review]
LGTM 👍. Thanks, @csi-lk!

**Comment by csi-lk** [comment]
@kevin940726 feel free to merge (I don't have write access)

**Pull request merged**